.babelrc

```diff
@@ -1,3 +1,2 @@
 {
-  "stage": 0
```

Comment:
why did you remove this? I guess it's not needed anymore but did you ran into an issue?